update.php

```diff
@@ -817,3 +817,13 @@ function update_1400()
 
 	return Update::SUCCESS;
 }
+
+function pre_update_1402()
+{
+	// Necessary before a primary key change
+	if (!DBA::e("TRUNCATE TABLE `parsed_url`")) {
```

Comment:
I guess it would be better to use  `DROP TABLE`here.

Reply:
I tested the update successfully so I limited this query but it's fine with me as well.